docs/community-ee/community-ee-release-process.md

```diff
@@ -11,68 +11,65 @@ The EE versioning convention is core tag plus patch number, for example:
     - EE with core `2.16.2` comes out -> `community-ee:2.16.2-1`
     - EE with core `2.16.3` comes out -> `community-ee:2.16.3-1`
 
-## Dependencies
-
-- Podman/Docker
-
 ## Credentials
 
-- Access to the https://github.com/ansible-community/images repository
-- Access to ghcr.io
+- Access to https://github.com/ansible-community/images repo.
+- Join [Release Management working Matrix room](https://app.element.io/?updated=1.11.38#/room/#release-management:ansible.com) and [Execution Envs group in Forum Group](https://forum.ansible.com/g/ExecutionEnvs).
+- Access to Ansible Release Management Group in Github.
+- Access to [eercheck](https://github.com/anweshadas/eercheck) repo.
 
 ## Prerequisites
 
 - Join the [Release Management working group](https://forum.ansible.com/g/release-managers) and [Execution Environment group](https://forum.ansible.com/g/ExecutionEnvs).
 - Understand [Ansible execution environments](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341).
 - Read about the [eerelease.yml](/.github/workflows/eerelease.yml)  GitHub workflow.
 - Show intention and book the date for the releasing of the EE.
+-  Prepare the release announcments - both email templates and matrix communication draft ready.
```

Comment:
```suggestion
- Prepare the release announcement drafts for the emails and matrix communication.
```

Reply:
There is no email template. It's a forum post and a matrix post.